bundle/config/mutator/normalize_paths.go

```diff
@@ -0,0 +1,80 @@
+package mutator
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"path/filepath"
+
+	"github.com/databricks/cli/bundle"
+	"github.com/databricks/cli/bundle/config/mutator/paths"
+	"github.com/databricks/cli/libs/diag"
+	"github.com/databricks/cli/libs/dyn"
+)
+
+type normalizePaths struct{}
+
+func (a normalizePaths) Name() string {
+	return "NormalizePaths"
+}
+
+func NormalizePaths() bundle.Mutator {
```

Comment:
Please include a comment explaining the pre- and post-conditions.

Also good to include this is only applied for resources that were loaded from YAML.